ding_place2book.install

```diff
@@ -64,6 +64,7 @@ function ding_place2book_schema() {
  * Implementation of hook_install().
  */
 function ding_place2book_install() {
+  variable_set('ding_place2book_event_nodes', array('defaults' => array('maintain_copy' => 'maintain_copy')));
```

Comment:
It would be nice with a comment about why this variable is set!
Should it be set in an update function as well?